Documentation/final-report/pages/technical_solution/machine_learning.tex

```diff
@@ -770,12 +770,11 @@ \subsubsection{Summary of main challenges}
 
 The biggest initial
 hurdle was finding a labelling software to annotate our satellite images.
-Initially, we had settled on LabelImg, however it was no longer being supported
-on most of our operating systems and was crashing on launch. Instead, we turned
-towards Label Studio and the ML Backend extension, which we intended on using
+Initially, we turned towards Label Studio and the ML Backend extension, 
```

Comment:
We started with Label Studio, a popular labelling software with their integrated ML backend extension